nio/client/async_client.py

```diff
@@ -464,6 +466,19 @@ async def create_matrix_response(
             parsed_dict = await self.parse_body(transport_response)
             resp = DeleteDevicesAuthResponse.from_dict(parsed_dict)
 
+        elif issubclass(response_class, RoomGetEventResponse):
```

Comment:
I don't think you need the whole elif here, this can be handled in the else branch further bellow?

Reply:
Done